src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -48,30 +49,21 @@
 
     public static final double DEFAULT_TARGET_FACTOR_THRESHOLD_PERCENTILE = 25.0;
 
+    public static final double DEFAULT_MAXIMUM_PERCENT_ZEROS_IN_COLUMN = 2.0;
+
+    public static final double DEFAULT_MAXIMUM_PERCENT_ZEROS_IN_TARGET = 5.0;
+
     public static final double DEFAULT_COLUMN_OUTLIER_DROP_THRESHOLD_PERCENTILE = 2.5;
 
     // 0.1% by default as copied from python code... perhaps a bug there as most of these constants in that code
     // are then multiplied by 100 or subtracted from a 100, check out PanelCleaner.py in ReCapSeg repo.
     public static final double DEFAULT_OUTLIER_TRUNCATE_PERCENTILE_THRESHOLD = 0.1;
 
-    public static final double DEFAULT_MAXIMUM_PERCENT_ZEROS_IN_COLUMN = 2.0;
-
-    public static final double DEFAULT_MAXIMUM_PERCENT_ZEROS_IN_TARGET = 5.0;
-
     public static final String INFER_NUMBER_OF_EIGEN_SAMPLES = "auto";
```

Comment:
Probably not worth fixing since the new coverage model won't have eigensamples or eigen samples.